test/chunking-form/samples/namespace-object-import-es3/_config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
```

Comment:
As there are no chunks involved, it might make sense to move this test into the `form` category (which should produce slightly nicer error diffs).

Reply:
Will move to `forms` as recommended. Do you have any preference for test name?

Reply:
No preference, the name is totally fine!